### PR TITLE
OCP 4.17 Release Notes for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.17

Issue: 
- https://jsw.ibm.com/browse/POCPQE-81

Link to docs preview:
- P: https://80276--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-ibm-power_release-notes
- Z: https://80276--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-ibm-z_release-notes

Additional information:
- Review draft: https://docs.google.com/document/d/1O6ZKXmpkHpoO1opyD9ohdq5YsOXp8GHqPDeDGMFbIyc/edit

QE review:
- For P : Manoj Kumar, Luvella , Julie, Sajauddin, Paul
- for Z : Holger Wolf, Florian Leber
- [x] QE has approved this change.

Branch : enterprise-4.17
@SNiemann15

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -104,6 +118,14 @@ Compute nodes must run {op-system-first}.
 
 The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
 
+This release introduces support for the following features on {ibm-power-name}:
+
+* Add cluster-etcd-operator support for additional slower hardware conditions
+* Compliance Operator DISA-STIG
```
Comment (prb112) [inline release_notes/ocp-4-17-release-notes.adoc:124]:
```suggestion
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -104,6 +118,14 @@ Compute nodes must run {op-system-first}.
...
+* Installer Provisioned Infrastructure for IBM PowerVS - move to CAPI
+* Secondary Scheduler Operator
+* Scheduler Plugins
```
Comment (prb112) [inline release_notes/ocp-4-17-release-notes.adoc:127]:
```suggestion
```

Comment (alishaIBM) [reply release_notes/ocp-4-17-release-notes.adoc]: Ack. Updated the PR.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (alishaIBM) [reply release_notes/ocp-4-17-release-notes.adoc]: Ack. Updated the PR.

**Review by SNiemann15 — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -85,6 +85,20 @@ xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing
...
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* CPU manager
+* Compliance Operator DISA-STIG
+* Non-volatile Memory Express (NVMe) support for LPAR
+* Secondary Scheduler Operator
+* Scheduler Plugins
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply release_notes/ocp-4-17-release-notes.adoc:100]: This is the correct wording for etcd see https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/release_notes/ocp-4-14-release-notes#ocp-4-14-tuning-etcd-latency-tolerances
```suggestion
* Tuning etcd latency tolerances 
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 4]
```diff
@@ -104,6 +118,11 @@ Compute nodes must run {op-system-first}.
 
 The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
 
+This release introduces support for the following features on {ibm-power-name}:
+
+* Add cluster-etcd-operator support for additional slower hardware conditions
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply release_notes/ocp-4-17-release-notes.adoc:123]: Verified with Laura, this is the correct wording for etcd see https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/release_notes/ocp-4-14-release-notes#ocp-4-14-tuning-etcd-latency-tolerances
```suggestion
* Tuning etcd latency tolerances 
```

**Review by prb112 — APPROVED** [review]
/lgtm

**Comment by holgwolf** [comment]
LGTM

File: release_notes/ocp-4-17-release-notes.adoc [hunk 5]
```diff
@@ -85,6 +85,18 @@ xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing
...
+[discrete]
+[id="ocp-4-17-ibm-z-enhancements_{context}"]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* CPU manager
+* Non-volatile Memory Express (NVMe) support for LPAR
```
Comment (lahinson) [inline release_notes/ocp-4-17-release-notes.adoc:97]:
```suggestion
* Non-volatile memory express (NVMe) support for LPAR
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 6]
```diff
@@ -104,6 +116,11 @@ Compute nodes must run {op-system-first}.
 
 The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
 
+This release introduces support for the following features on {ibm-power-name}:
```
Comment (lahinson) [inline release_notes/ocp-4-17-release-notes.adoc:119]:
```suggestion
This release introduces support for the following features on {ibm-power-title}:
```

**Review by lahinson — COMMENTED** [review]
Just a couple of minor things to fix. Otherwise, LGTM.

**Comment by alishaIBM** [comment]
@lahinson thank you for the review. I have incorporated the changes.

/label merge-review-needed

**Review by bergerhoffer — COMMENTED** [review]
LGTM for merging

**Pull request merged**